src/components/photoUpload/PhotoUpload.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import type {
+  RcFile,
+  UploadChangeParam,
+  UploadFile,
+  UploadProps,
+} from "antd/es/upload/interface";
+import { message, Upload } from "antd";
+import ImgCrop from "antd-img-crop";
+import { LoadingOutlined, UserOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import {
+  UPLOADING,
+  DONE,
+  ERROR,
+  IS_JPEG,
+  IS_PNG,
+} from "utils/photoUploadConsts";
+
+function PhotoUpload() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [imageUrl, setImageUrl] = useState<string>(
+    "https://pbs.twimg.com/profile_images/1541920820204797952/RaMNiqHx_400x400.jpg"
```

Comment:
never use url in the code. move to env or to constants

Reply:
Removed. Sorry, I forgot to remove, this was for a test